src/runtime_src/core/edge/drm/zocl/zocl_kds.c

```diff
@@ -283,11 +283,26 @@ int zocl_add_context_kernel(struct drm_zocl_dev *zdev, void *client_hdl, u32 cu_
 	int ret = 0;
 	struct kds_ctx_info info = { 0 };
 	struct kds_client *client = (struct kds_client *)client_hdl;
+	struct kds_client_ctx *cctx = NULL;
+
```

Comment:
It's better to check whether any existing context is already present or not. 
Otherwise, it will end up with duplicate context. 
I am fine if this is a temporary fix. We can add the check later also.